common/src/main/java/com/wynntils/services/mapdata/attributes/type/MapAttributes.java

```diff
@@ -33,6 +33,9 @@ public interface MapAttributes {
     // If this is the empty string (""), then no label will be displayed
     Optional<String> getLabel();
 
+    // If this is the empty string (""), then no secondary label will be displayed
```

Comment:
I'm thinking we can maybe find a better term than `secondaryLabel`. Perhaps `description` or `info` or something like that? I'd like for the basic attributes to have really good names: short, succinct, distinct and clear. Maybe we can do no better than "secondaryLabel" but I think it could be worth thinking a bit more about, since once we release MapData this cannot ever be changed.

Reply:
Maybe `description` then? It sounds like it is inviting some extended information that you can get if you request it.